integration/app/app.component.ts

```diff
@@ -1,13 +1,24 @@
 import { Component, ViewEncapsulation } from '@angular/core';
 import { Store, Select } from '@ngxs/store';
+import { UpdateFormDirty } from '@ngxs/form-plugin';
 import { Observable } from 'rxjs/Observable';
-
-import { AddTodo, RemoveTodo, TodoState } from './todo.state';
+import { FormBuilder } from '@angular/forms';
+import { AddTodo, RemoveTodo, TodoState, SetPrefix } from './todo.state';
 
 @Component({
   selector: 'app-root',
   template: `
     <div class="todo-list">
+      <div>
+        <h3>Reactive Form</h3>
+        <form [formGroup]="pizzaForm" novalidate (ngSubmit)="onSubmit()" ngxsForm="todos.pizza">
+            <input type="text" formControlName="toppings" />
+            <br><hr>
+            <button type="submit">set Dirty false</button>
+            <button type="button" (click)="onPrefix()">Set Prfix</button>
+        </form>      
```

Comment:
Lint is complaining about the trailing whitespace on line 19. Just need to remove it.